cocos/2d/utils/text-utils.ts

```diff
@@ -26,7 +26,7 @@
 
 import { RUNTIME_BASED } from 'internal:constants';
 import { minigame } from 'pal/minigame';
-import Pool from '../../core/utils/pool';
+import { js } from '../../core';
```

Comment:
```suggestion
import { js } from '../../core';
const { Pool } = js;
```

Reply:
then we don't need to change the code `new Pool()` to `new js.Pool()`

Reply:
I don't think there is any benefit about it. And use js.Pool make it more clearly, as core also exports Pool too.